protocols/delegate/contracts/Delegate.sol

```diff
@@ -137,15 +108,14 @@ contract Delegate is IDelegate, Ownable {
     * @param _priceCoef uint256 The whole number that will be multiplied by 10^(-priceExp) - the price coefficient
     * @param _priceExp uint256 The exponent of the price to indicate location of the decimal priceCoef * 10^(-priceExp)
     */
-  function setRule(
+  function _setRuleInternal(
     address _senderToken,
     address _signerToken,
     uint256 _maxSenderAmount,
     uint256 _priceCoef,
     uint256 _priceExp
-  ) external onlyAdmin {
-
-    rules[_senderToken][_signerToken] = Rule({
+  ) internal {
```

Comment:
Convention tends to be to put `internal` functions at the end of the contract, and `external`/`public` functions beforehand

Reply:
really? I thought it was easier to read since the methods were visually grouped.

Reply:
Yeah I do see your point. It's the convention I've known since before I was at airswap, and I believe it's true of all the airswap contracts too.

Reply:
I think we were doing it with crev2, but I wasn't a fan since I was jumping all over the file. If you think it should be ordered as such I'll do it.

Reply:
Discussed with Don and this is the convention he uses too, so we will move the functions to the end.